horiz_interp/horiz_interp_type.F90

```diff
@@ -189,35 +189,35 @@ subroutine horiz_interp_type_eq(horiz_interp_out, horiz_interp_in)
       call mpp_error(FATAL,'horiz_interp_type_eq: horiz_interp_type variable on right hand side is unassigned')
     endif
 
-    horiz_interp_out%faci            => horiz_interp_in%faci
```

Comment:
The functionality of this routine is changing.  Can you add doxygen comments to this clearly stating that this routine creates a *copy* of the object.  Also doxygen comments for the variables.

Reply:
Let me know if these changes are sufficient